### PR TITLE
Alternative base structure.

### PR DESCRIPTION
**Pull request by mariorez**

### CODE REVIEW

**Comment by hussani** [comment]
:+1:
Make more sense.

**Comment by xshifty** [comment]
:+1:
I line this way too. But, where we'll put the tests?

**Comment by mariorez** [comment]
I put the directory "Tests" inside "src" and for the "Tests" classes I use the same namaspace of domain classes. Thus it is easier to import the dependencies in the Tests classes. Ex:
```php
/* /src/Domain/Task.php */
namespace DddFellowship\Domain\Task
class Task { }

/* /src/Test/TestTask.php */
namespace DddFellowship\Domain\Task
class TestTask { }
```

**Comment by xshifty** [comment]
@ddd-fellowship/owners Someone else want to vote?

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     "require-dev": {
         "phpunit/phpunit": "*"
     },
     "autoload": {
         "psr-4": {
-            "DddFellowship\\TaskManager\\": "src/",
-            "DddFellowship\\TaskManager\\Test\\": "test/"
+            "DddFellowship\\View\\":          "src/View/",
+            "DddFellowship\\Application\\":   "src/Application/",
+            "DddFellowship\\Domain\\":        "src/Domain/",
```
Comment (codeliner) [inline composer.json]: The Domain is the problem space and the Model the solution space so this should be called Model not Domain

**Comment by codeliner** [comment]
:-1: for putting tests under same path like production code. Normally you use the composer "autoload-dev" option to load tests only in dev mode.

Not sure about removing the TaskManager from namespace. It is the module name. What if we want to split the app in different modules later?

**Comment by xshifty** [comment]
@codeliner maybe is time for us discuss about inicial structure.

I think DddFellowship\TaskManager must be ower top level namespace.

About autoload-dev, i didn't know about this. Thank you.

I'll create a new issue.

**Pull request merged**